Updates to status names (#12)
twitter/birdwatch: content/ranking-notes.md

katex: true
---

> Birdwatch notes are submitted and rated by contributors. Ratings are used to determine note status labels (“Helpful”, “Not Helpful”, or “Needs More Ratings”). Note statuses also determine which notes are displayed on each of the [Birdwatch Site’s timelines](../birdwatch-timelines/), and which notes are displayed as [Birdwatch Cards](../notes-on-twitter/) on Tweets.

Reviewer: ```suggestion
Birdwatch notes are submitted and rated by contributors. Ratings are used to determine note statuses (“Helpful”, “Not Helpful”, or “Needs More Ratings”). Note statuses also determine which notes are displayed on each of the [Birdwatch Site’s timelines](../birdwatch-timelines/), and which notes are displayed as [Birdwatch Cards](../notes-on-twitter/) on Tweets.
```

All Birdwatch notes start out with the Needs More Ratings status until they receive at least 5 total ratings. Once a note has received at least 5 ratings, it is assigned a Note Helpfulness Score according to the algorithm described below.

If a note is deleted, the algorithm will still score it (using all non-deleted ratings of that note) and the note will receive a status if it’s been rated more than 5 times, although since it is deleted it will not be shown on Twitter even if its status is Helpful.

> Notes with a Note Helpfulness Score of 0.40 and above earn the status of Helpful. Notes with a Note Helpfulness Score less than -0.05 -0.8 \* abs(noteFactorScore) are assigned Not Helpful, where noteFactorScore is described in [Matrix Factorization](#matrix-factorization). Notes with scores in between remain labeled as Needs more Ratings.

Reviewer: ```suggestion
Notes with a Note Helpfulness Score of 0.40 and above earn the status of Helpful. Notes with a Note Helpfulness Score less than -0.05 -0.8 \* abs(noteFactorScore) are assigned Not Helpful, where noteFactorScore is described in [Matrix Factorization](#matrix-factorization). Notes with scores in between remain with a status of Needs more Ratings.
```

> In addition to Helpful / Not Helpful statuses, labels also show the two most commonly chosen explanation tags which describe the reason the note was rated helpful or unhelpful.

Reviewer: ```suggestion
When a note reaches a status of Helpful / Not Helpful, Birdwatch also shows the two most commonly chosen explanation tags which describe the reason the note was rated helpful or unhelpful.
```

The resulting scores that we use for each note are the note intercept terms i_n. These scores on our current data give an approximately Normal distribution, where notes with the highest and lowest intercepts tend to have factors closer to zero.

> We currently set the thresholds to achieve a “Helpful” label at 0.40, including less than 10% of the notes, and our threshold to achieve a “Not Helpful” label at –0.08. However, these are far from set in stone and the way we generate status labels from note scores will evolve over time.

Reviewer: ```suggestion
We currently set the thresholds to achieve a “Helpful” status at 0.40, including less than 10% of the notes, and our threshold to achieve a “Not Helpful” status at –0.08. However, these are far from set in stone and the way we generate statuses from note scores will evolve over time.
```

### Determining Note Status Explanation Tags

> When notes are labeled with a status (either Helpful or Not Helpful), we display the top two explanation tags that were given by raters to explain why they rated the note helpful or not.

Reviewer: ```suggestion
When notes reach a status of Helpful or Not Helpful, Birdwatch displays the top two explanation tags that were given by raters to explain why they rated the note helpful or not.
```

> This is done by simply counting the number of times each explanation tag was given filtered to explanation tags that match the final note status label (e.g., if the note status is Helpful we only count helpful explanation tags). Importantly, each explanation tag must be used by at least two different raters. If there aren’t two different tags that are each used by two different raters, then the note’s status label is reverted back to “Needs More Ratings” (this is very rare).

Reviewer: ```suggestion
This is done by simply counting the number of times each explanation tag was given filtered to explanation tags that match the final note status (e.g., if the note status is Helpful we only count helpful explanation tags). Importantly, each explanation tag must be used by at least two different raters. If there aren’t two different tags that are each used by two different raters, then the note’s status is reverted back to “Needs More Ratings” (this is very rare).
```
